package.json

```diff
@@ -48,7 +49,8 @@
     ]
   },
   "resolutions": {
-    "@types/react": "17.0.47",
+    "@types/react": "16.14.0",
+    "@types/react-dom": "16.9.0",
```

Comment:
There is no `16.14.x` for `@types/react-dom`. This is the latest minor in 16.

Reply:
Hmm, does this mean we need to do additional testing?